frontend/src/services/authProvider.jsx

```
@@ -17,6 +17,14 @@ const authReducer = (state, action) => {
             return { ...state, userInfo: action.payload };
         case 'LOGIN_AND_SET_USER_INFO':
             return { isLoggedIn: true, userInfo: action.payload };
+        case 'UPDATE_AND_SET_UPDATED_USER_INFO':
+            let updatedUserInfo = {
+                ...state.userInfo,
+                ...action.payload,
+            };
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)